### ISSUE TITLE
Jittery movement on the hoverbike

### ISSUE THREAD

**Issue by ghost** [fgsfdsfgs/perfect_dark#78]
Sometimes movement on the Hoverbike will jitter, especially with camera rotation.

https://github.com/fgsfdsfgs/perfect_dark/assets/130617241/34c694af-060a-4d9a-aebf-db68801ba8fc

**Comment by ghost** [comment]
I could not replicate this using a controller. Perhaps the mouse look and the bike automatically adjusting player view are conflicting?

**Comment by MultipleMegaPixels** [comment]
This happened to me with the slow motion cheat, but it was even worse

**Comment by ghost** [comment]
> This happened to me with the slow motion cheat, but it was even worse

(Epilepsy warning)

https://github.com/fgsfdsfgs/perfect_dark/assets/130617241/ec4f34b4-7c3f-4a09-bb6f-81597e3b9ec6

**Comment by MultipleMegaPixels** [comment]
Yes, like this!

> > This happened to me with the slow motion cheat, but it was even worse
> 
> (Epilepsy warning)
> 
>  0278-1554.mp4

**Comment by mmillar-bolis** [comment]
This is happening for me on Windows 10 while using either a controller or a mouse. In fact, if I just shove the bike round, the bike seems to occasionally jitter all by itself, whether I am on it or not.

If I cap framerate to 30fps alone, the issue still occurs. However, if I cap framerate *and* disable v-sync, the jittering problem no longer occurs.

EDIT: On my system, if I force v-sync through GPU driver settings, the issue will happen regardless of whether I enable or disable v-sync in the `.ini`. Probably an unnecessary detail, but I thought I'd mention it.

**Comment by ghost** [comment]
Also to note is that the Hover bike works fine on 1964 GEPD with slow motion and 60 FPS, if it was something collision tested related it would also break here.

https://github.com/fgsfdsfgs/perfect_dark/assets/130617241/2c019a25-f821-4552-836c-7d474453b424

**Comment by mmillar-bolis** [comment]
In 'Area 51: Escape', the Elvis hoverbed jitters as well. It's especially noticeable near the beginning when you need to hide Elvis before going to meet Jonathan.

**Comment by rafccq** [comment]
Guys, please help test this fix if you can. You can grab it [here](https://github.com/fgsfdsfgs/perfect_dark/actions/runs/6226962242), thanks!

**Comment by mmillar-bolis** [comment]
@rafccq I just tried your build for Windows. For fun, I tested both with and without v-sync. Riding on and pushing around the hover bike on the CI hub map, as well as pushing Elvis around on the bed in A51: Escape, and the jitters appear to be resolved. I used the slow-mo cheat on A51: Infiltration and rode around on the hover bike, and the jitters appear to be resolved there as well.

Great work!!
